modules/support-reviewing-an-access-request-from-an-email-notification.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-support.adoc
+// * support/getting-support.adoc
+// * service_mesh/v2x/ossm-troubleshooting-istio.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-an-email-notification_{context}"]
+= Reviewing an access request from an email notification
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+
+
+.Procedure
+
+. Click the link within the email to bring you to the {hybrid-console-second}.
+
+. Click *Approve* or *Deny* under *Decision* in the *Access Request Details* modal.
```

Comment:
*Deny* requires a justification which I see is already noted but not in these steps.

Reply:
added it in here too:
"[NOTE]
====
Denying an access request will require you to complete the *Justification* field. In this case, SRE will not be able to directly act on the resources related to the incident. Customers can still use the link:https://access.redhat.com/support/cases/#/case/list[*Customer Support*] to help investigate and resolve any issues.
===="